dev/NavigationView/NavigationView.cpp

```diff
@@ -365,14 +365,23 @@ void NavigationView::OnApplyTemplate()
     if (SharedHelpers::IsThemeShadowAvailable())
     {
 #ifdef USE_INSIDER_SDK
-        if (auto splitView = m_rootSplitView.get())
+        // Shadow will get clipped if casting on the splitView.Content directly
+        // Creating a canvas with negative margins as receiver to allow shadow to be drawn outside the content grid 
+        winrt::Canvas shadowReceiver;
```

Comment:
Why not update the template to include this?

Reply:
I was thinking if people re-templated then they won't get shadow. I guess I can put it in template as long as we are ok with that.

Reply:
Good point about re-templating. Since we generally want shadows to work on existing apps it makes sense to keep this in code.